### DISCUSSION TITLE
Category field - Allow to select categories from multiple groups

### DISCUSSION THREAD

**Discussion by Djive** [craftcms/cms#8901] [Ideas] [answered]
### Description

Can you enhance category field 
![category-multiple-groups](https://user-images.githubusercontent.com/1241189/90343211-84c6b380-e00e-11ea-8d85-885680db2c14.jpg)

**Comment by brandonkelly** [comment, marked as answer]
This isn’t currently possible for us to add, but will be in a future version.
